answers/templates/queue.html

```diff
@@ -64,16 +64,21 @@ <h1>{{ hunt_name }} - Answer Queue</h1>
 
 <script type="text/javascript">
 $(document).ready(function() {
+    var queue_table = $('#queue')
+    var queue_table_dom = queue_table.get(0);
+
     // For choice fields, each entry should only be searchable by the currently
     // selected value.
     $.fn.dataTableExt.ofnSearch['select-input'] = function(value) {
-        return $(value).find('select').val();
+        var cell = queue_table_dom.rows[i + 1].cells[j];
```

Comment:
Do you need update line 72 to `function(i, j)`?

Reply:
Done.